src/views/graphs.js

```diff
@@ -352,7 +352,7 @@ class Graphs extends Component {
                         <NavLink
                             className={classnames("py-2 px-3", {
                             // TODO(tdavis): Make this actually toggle properly
```

Comment:
Could you delete this todo statement, since toggling is fixed?

Reply:
Done!